org.eclipse.xtext.xbase.lib/build.gradle

```diff
@@ -5,3 +5,23 @@ description = 'The runtime library for Xbase languages.'
 dependencies {
 	compile "com.google.guava:guava:$versions.guava"
 }
+
+task copyGwtSources(dependsOn: 'build') {
+	doLast {
+	
+		ant.copy(toDir: '../org.eclipse.xtext.xbase.lib.gwt/build/gwtSources', verbose: true) {
```

Comment:
[Copy](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Copy.html) has quite a lot of options. And it's no problem to add a `doLast` block with some custom code (that doesn't overwrite anything from the superclass).